dist/cms.js

```diff
@@ -4,6 +4,39 @@ function usesBlockTheme() {
   return !!document.querySelector('div.wp-site-blocks');
 }
 
+/**
+ * Detects the WordPress parent and child theme slugs.
+ *
+ * See https://core.trac.wordpress.org/changeset/59698.
+ *
+ * @returns {object} Object with fields `theme` and `parent_theme`.
+ */
+function getWordPressTheme() {
+  const theme = {
+    theme: 'unknown',
+    parent_theme: '',
+  };
```

Comment:
From a tracking perspective, I think it might be better to go with `theme` and `child_theme` (or explicitly `parent_theme` and `child_theme`), in line with how these classes are added.

The conditional approach you're currently using is most common in WordPress terminology, but with that approach any theme that is not made as a child theme could appear under either `theme` or `parent_theme`, so it makes tracking more difficult than needed.

So I think we should always put the parent theme and child theme under the same respective property.

Reply:
> The conditional approach you're currently using is most common in WordPress terminology, but with that approach any theme that is not made as a child theme could appear under either `theme` or `parent_theme`

True. I intended for this to make it easier to track a specific theme being used (simply always check `theme.theme`, but I didn't fully think it through.

But I guess if you want to know how often a specific theme is actually used (without child themes), then one could do `theme === xyz && child_theme === ''``